redash/destinations/webhook.py

```diff
@@ -36,7 +36,8 @@ def notify(self, alert, query, user, new_state, app, host, options):
             data = {
                 'event': 'alert_state_change',
                 'alert': serialize_alert(alert, full=False),
-                'url_base': host 
+                'url_base': host,
+                "description": alert.render_template() if alert.template else ''
```

Comment:
Let's encapsulate this logic in the `render_template` function, i.e. just return an empty string if the template is empty.

Reply:
refactored. thanks!